components/support/rc_crypto/nss/src/ec.rs

```diff
@@ -342,6 +349,7 @@ impl PublicKey {
 fn check_pub_key_bytes(bytes: &[u8], curve: Curve) -> Result<()> {
     let field_len = match curve {
         Curve::P256 => 32,
+        Curve::SECP384R1 => 32,
```

Comment:
Ditto, maybe factorize this operation on the `Curve` object.

Reply:
done